koku/cost_models/test/tests_serializers.py

```diff
@@ -363,11 +357,11 @@ def test_get_metric_display_data_openshift(self):
         """Test the display data helper function for OpenShift metrics."""
         serializer = CostModelSerializer(data=None)
 
-        for metric_choice in CostModelMetricsMap.METRIC_CHOICES:
+        for metric_choice in constants.METRIC_CHOICES:
             response = serializer._get_metric_display_data(Provider.PROVIDER_OCP, metric_choice[0])
-            self.assertIsNotNone(response.label_measurement_unit)
-            self.assertIsNotNone(response.label_measurement)
-            self.assertIsNotNone(response.label_metric)
+            self.assertIsNotNone(response.get("label_measurement_unit", None))
```

Comment:
None is the default return value for .get() so you don't actually need to include it.

Reply:
updated

Reply:
updated